src/earthkit/meteo/score/kge.py

```diff
@@ -0,0 +1,18 @@
+# (C) Copyright 2025 ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+#
+
+from . import array
+
+
+def kge(*args, **kwargs):
+    return array.kge(*args, **kwargs)
+
+
+def kge_prime(*args, **kwargs):
```

Comment:
@andreas-grafberger modified_kge instead? Can you ask the hydro people for a good name? I feel that prime won't be clear enough.